build.sbt

```diff
@@ -51,20 +31,23 @@ developers := List(
 licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt"))
 
 resolvers ++= Seq(
-  "bintray/non" at "http://dl.bintray.com/non/maven",
-  "bintray/scalaz" at "http://dl.bintray.com/scalaz/releases"
+  "bintray/non" at "http://dl.bintray.com/non/maven"
 )
 
 scalacOptions ++= Seq(
   "-deprecation",
   "-feature"
 )
 
-val JawnVersion = "0.8.4"
+val JawnVersion = "0.10.1"
+
+val Fs2Version = "0.9.1"
 
 libraryDependencies ++= Seq(
   "org.spire-math" %% "jawn-parser" % JawnVersion,
-  "org.spire-math" %% "jawn-ast" % JawnVersion % "test",
-  "org.scalaz.stream" %% "scalaz-stream" % scalazStreamVersion(scalazVersion.value),
-  "org.specs2" %% "specs2-core" % specs2Version(scalazVersion.value) % "test"
+  "org.scodec"     %% "scodec-bits" % "1.1.1",
```

Comment:
I think this, and the `ByteVector` instance, can go away.  It's a great library, but `ByteVector` is not as necessary in an fs2 world with its improved chunking support.